### PR TITLE
Update contribution guide and Issue/PR templates

### PR DESCRIPTION
**Pull request by paulbalaji**
Signed-off-by: Paul Balaji <paulbalaji@improbable.io>

**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
links in contributions guide and issue/PR templates now link to main gdk repo
#### Tests
How did you test these changes prior to submitting this pull request?
What automated tests are included in this PR?
#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

**Comment by paulbalaji** [comment]
mirrors: https://github.com/spatialos/gdk-for-unity-blank-project/pull/3

**Comment by paulbalaji** [comment]
Part 2 of 3
https://github.com/spatialos/gdk-for-unity/pull/571
https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/54
https://github.com/spatialos/gdk-for-unity-blank-project/pull/3

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,23 +1,10 @@
...
-later releases and we’ll keep you posted on this). However, we are taking issues and we do want your 
-[feedback](../README.md#give-us-feedback).
-
-* **Improbable developers**: <br/>
-See the [Contributions guide](https://improbableio.atlassian.net/wiki/x/foDrDw).
+We are currently not accepting public contributions (but we do plan to in later releases and we’ll keep you posted on this). However, we are taking issues and we do want your [feedback](../README.md#give-us-feedback).
 
 ## Coding standards
-See the [Unity GDK C# coding standards guide](../docs/contributions/unity-gdk-coding-standards.md).
+See the [GDK for Unity C# coding standards guide](../coding-standards.md).
```
Comment (jessicafalk) [inline .github/CONTRIBUTING.md]: link

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,23 +1,10 @@
...
 
 ## Getting started
-See [About the SpatialOS Unity GDK](../docs/README.md) for
- details on how to get started using the Unity GDK.
-
-## Warning
-This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation -
- see the guidance on
-  [Recommended use](../README.md#recommended-use).
+See [About the SpatialOS GDK for Unity](https://docs.improbable.io/unity/alpha/get-started) for details on how to get started using the Unity GDK.
```
Comment (jessicafalk) [inline .github/CONTRIBUTING.md]: reword sentence to 

See [the Get started guide](https://docs.improbable.io/unity/alpha/get-started) for details on how to use the SpatialOS GDK for Unity

Comment (jessicafalk) [reply .github/CONTRIBUTING.md]: same for the other one that already got merged in :P

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](https://github.com/spatialos/gdk-for-unity#give-us-feedback).
+**Contributions**: We are not currently taking public contributions - see our [contributions](../blob/master/.github/CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](https://github.com/spatialos/gdk-for-unity#give-us-feedback).
```
Comment (jessicafalk) [inline .github/pull_request_template.md]: link doesn't seem to work

Comment (paulbalaji) [reply .github/pull_request_template.md]: the link works when in actual PRs - which is what the template is for

**Pull request merged**